common/config/rush/common-versions.json

```diff
@@ -24,7 +24,7 @@
      * instead of the latest version.
      */
     // "some-library": "1.2.3"
-    "@azure/communication-calling": "1.4.3-beta.1"
+    "@azure/communication-calling": "1.5.4-beta.1 || 1.4.4"
```

Comment:
Not sure about my update here but when I tried to have the version here as:
```
"@azure/communication-calling": "1.5.4-beta.1"
```
and I get this error when I run `rush update` or `rush update --full`
![mis-matching dependencies](https://user-images.githubusercontent.com/79475487/169639148-7d8a1a90-dde2-4b89-a896-d624f9fbf3ea.png)

Reply:
This isn't right. This should specify just the version we want for beta.

Instead, update line 55. That should make the warning disappear.